Pearson-Advance/frontend-app-skillable
# PADV-1474 feat: frontend - Create Gauge charts.

## Tickets

- https://agile-jira.pearson.com/browse/PADV-1474

## Description
This PR adds a React component named LabDetailsChartCard to display detailed information using gauge charts. The component should fetch data from the /details endpoint and use the MUI Charts library for rendering. The visual layout should be similar to the provided design, with two separate gauge charts representing tasks and scores.

## Changes Made

- [x] Updating react and react-dom to version 17
- [x] Add dependencies to mui charts: @mui/material @emotion/react @emotion/styled
- [x] Add @mui/x-charts library.
- [x] Add component LabDetailsChartCard to the LabDetails view.

## How To Test

Before any prior step, go to your MFE folder and execute `npm i` to install the new dependencies.

1. Please have your devstack environment running, run `npm start` and the MFE shall be running in port 2003 
2. go to the LMS and access to a skillable course.
3. Click on Training Lab tab to access the MFe.
4. Navigate from Strudent Roster to a Lab instance detail.
5. Verify the Gauge chart card.

![Screenshot from 2024-08-08 15-59-41](https://github.com/user-attachments/assets/f217a098-dc2a-40fd-877d-063e0e025399)
